larq/quantizers_test.py

```diff
@@ -400,7 +382,6 @@ def tanh_grad(x):
         ("magnitude_aware_sign", lq.quantizers.MagnitudeAwareSign),
         ("ste_tern", lq.quantizers.SteTern),
         ("dorefa_quantizer", lq.quantizers.DoReFa),
-        ("dorefa_kernel_quantizer", lq.quantizers.DoReFaKernel),
```

Comment:
If you still want to test this, I think we can use something like:
```python
("dorefa_kernel_quantizer", functools.partial(lq.quantizers.DoReFa, mode="weights")),
```
Otherwise, I think it is also fine to not add it here.

Reply:
Added testcase in b4bfd473f3ca7878dd10b6122a9d386404f73d89

Added
```
("dorefa_quantizer", functools.partial(lq.quantizers.DoReFa, mode="weights")),
```

because the quantizer string needs to be a valid one, because the test routine passes the quantizer name for some layers, which try to retrieve a valid class for the name string.

Please check!